setup.cfg

```diff
@@ -2,3 +2,9 @@
 with-doctest=1
 doctest-options=+ELLIPSIS,+IGNORE_EXCEPTION_DETAIL
 doctest-extension=rst
+with-coverage=1
+ignore-files=tube.py
```

Comment:
This line is because: https://github.com/blaix/tdubs/pull/14/commits/18d743e1a30f53b580fbb78e9c74e911be9d6ce2